issuetracker/factory/factory.go

```diff
@@ -3,6 +3,8 @@ package factory
 import (
 	"errors"
 
+	"github.com/preslavmihaylov/todocheck/issuetracker/internal/azureboards"
```

Comment:
You can move this line below the other "github.com/preslavmihaylov/todocheck/issuetracker/internal/...." imports to keep the import statements consistent.